# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/moz.build
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/moz.build
--- a/security/sandbox/moz.build	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/moz.build	Wed Jun 12 11:10:48 2019 +0100
@@ -47,15 +47,15 @@
         'chromium/base/debug/profiler.cc',
         'chromium/base/environment.cc',
         'chromium/base/hash.cc',
-        'chromium/base/lazy_instance.cc',
+        'chromium/base/lazy_instance_helpers.cc',
         'chromium/base/location.cc',
         'chromium/base/memory/ref_counted.cc',
         'chromium/base/memory/shared_memory_handle.cc',
         'chromium/base/memory/shared_memory_handle_win.cc',
         'chromium/base/memory/shared_memory_win.cc',
-        'chromium/base/memory/singleton.cc',
         'chromium/base/process/process_handle_win.cc',
         'chromium/base/rand_util_win.cc',
+        'chromium/base/scoped_clear_last_error_win.cc',
         'chromium/base/strings/nullable_string16.cc',
         'chromium/base/strings/string_number_conversions.cc',
         'chromium/base/strings/string_piece.cc',
@@ -67,10 +67,11 @@
         'chromium/base/strings/utf_string_conversions.cc',
         'chromium/base/synchronization/lock.cc',
         'chromium/base/synchronization/lock_impl_win.cc',
-        'chromium/base/third_party/dmg_fp/dtoa.cc',
+        'chromium/base/third_party/dmg_fp/dtoa_wrapper.cc',
         'chromium/base/third_party/dmg_fp/g_fmt.cc',
         'chromium/base/third_party/icu/icu_utf.cc',
         'chromium/base/third_party/superfasthash/superfasthash.c',
+        'chromium/base/threading/platform_thread.cc',
         'chromium/base/threading/platform_thread_win.cc',
         'chromium/base/threading/thread_collision_warner.cc',
         'chromium/base/threading/thread_id_name_manager.cc',
@@ -79,13 +80,17 @@
         'chromium/base/threading/thread_restrictions.cc',
         'chromium/base/time/time.cc',
         'chromium/base/time/time_win.cc',
+        'chromium/base/token.cc',
         'chromium/base/unguessable_token.cc',
+        'chromium/base/version.cc',
         'chromium/base/win/pe_image.cc',
         'chromium/base/win/scoped_handle.cc',
+        'chromium/base/win/scoped_handle_verifier.cc',
         'chromium/base/win/scoped_process_information.cc',
         'chromium/base/win/startup_information.cc',
         'chromium/base/win/windows_version.cc',
         'chromium/sandbox/win/src/acl.cc',
+        'chromium/sandbox/win/src/app_container_profile_base.cc',
         'chromium/sandbox/win/src/broker_services.cc',
         'chromium/sandbox/win/src/crosscall_server.cc',
         'chromium/sandbox/win/src/eat_resolver.cc',
@@ -100,6 +105,7 @@
         'chromium/sandbox/win/src/heap_helper.cc',
         'chromium/sandbox/win/src/interception.cc',
         'chromium/sandbox/win/src/interception_agent.cc',
+        'chromium/sandbox/win/src/ipc_args.cc',
         'chromium/sandbox/win/src/job.cc',
         'chromium/sandbox/win/src/named_pipe_dispatcher.cc',
         'chromium/sandbox/win/src/named_pipe_interception.cc',
@@ -128,6 +134,7 @@
         'chromium/sandbox/win/src/sandbox_policy_base.cc',
         'chromium/sandbox/win/src/sandbox_rand.cc',
         'chromium/sandbox/win/src/sandbox_utils.cc',
+        'chromium/sandbox/win/src/security_capabilities.cc',
         'chromium/sandbox/win/src/service_resolver.cc',
         'chromium/sandbox/win/src/sharedmem_ipc_client.cc',
         'chromium/sandbox/win/src/sharedmem_ipc_server.cc',